src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -925,16 +934,17 @@ public String getCallSettingsTypeName(Interface apiInterface) {
   }
 
   /** The name of the return type of the given grpc streaming method. */
-  public String getGrpcStreamingApiReturnTypeName(Method method, ImportTypeTable typeTable) {
+  public String getGrpcStreamingApiReturnTypeName(
+      MethodContext methodContext, ImportTypeTable typeTable) {
```

Comment:
It was an attempt to confine particular concepts to particular areas to reduce coupling. Maybe there really isn't enough value in being dogmatic in this case.